rm-datasource/src/main/java/io/seata/rm/datasource/ColumnUtils.java

```diff
@@ -155,12 +157,43 @@ public static String addEscape(String colName, Escape escape) {
         if (colName.charAt(0) == escape.value && colName.charAt(colName.length() - 1) == escape.value) {
             return colName;
         }
-        return String.format("%s%s%s", escape.value, colName, escape.value);
+
+        KeywordChecker keywordChecker = null;
+        if (escape == Escape.STANDARD) {
+            keywordChecker = KeywordCheckerFactory.getKeywordChecker(JdbcConstants.ORACLE);
+        } else if (escape == Escape.MYSQL) {
+            keywordChecker = KeywordCheckerFactory.getKeywordChecker(JdbcConstants.MYSQL);
+        }
```

Comment:
New continuous maintenance point, I hope there is a way to do it once and for all.
Such as passing dbType.